personal-tuya-devices/models/TS0601/_TZE200_rjxqso4a.yaml

```diff
@@ -0,0 +1,41 @@
+# https://community.smartthings.com/t/st-edge-personal-tuya-devices-generic-ef00-device/255270/468?u=w35l3y
+# _TZE200_rjxqso4a - carbon monoxide detector
+#[1, 'carbon_monoxide', tuya.valueConverter.trueFalse0],
+#[2, 'co', tuya.valueConverter.raw],
+#[9, 'self_test_result', tuya.valueConverter.selfTestResult],
+#[15, 'battery', tuya.valueConverter.raw],
+#[16, 'silence', tuya.valueConverter.raw],
+
+deviceLabel: Carbon Monoxide Detector
+profiles:
+  - child-carbonMonoxideDetector-v1
+  - child-carbonMonoxideMeasurement-v1
+  - normal-smokeDetector-v1
```

Comment:
As I mentioned before, you can't use `child-*` profiles in models.
They are reserved to generic mode.
Instead, duplicate `normal-smokeDetector-v1` and keep only the capabilities that exist in this model.
You may use the name `normal-coDetector-v1` for this new profile.
For example, `carbonMonoxideMeasurement` is not present in the  `normal-smokeDetector-v1` profile. Also `smokeDetector` is not present in this model.
Preferentially, in the same patch.

Reply:
This is done now. One more things is, I tried to change the profile to smoke detector in your main driver and this is the screen I am getting

<img width="274" alt="image" src="https://github.com/w35l3y/EdgeDrivers/assets/78273427/4ed8782e-2905-4b01-9585-6b1e7abd6c3b">

Reply:
This is because the profile `normal-smokeDetector-v1`has `smokeDetector` capability.
Your device doesn't, it has `carbonMonoxideDetector` and `carbonMonoxideMeasurement` instead.

https://github.com/w35l3y/EdgeDrivers/blob/f5e636f9749970fbf38f5b0d68c5e88c782ac8ff/personal-tuya-devices/profiles/normal-smokeDetector-v1.yaml#L8-L17

Reply:
> This is because the profile `normal-smokeDetector-v1`has `smokeDetector` capability. Your device doesn't, it has `carbonMonoxideDetector` and `carbonMonoxideMeasurement` instead.
> 
> https://github.com/w35l3y/EdgeDrivers/blob/f5e636f9749970fbf38f5b0d68c5e88c782ac8ff/personal-tuya-devices/profiles/normal-smokeDetector-v1.yaml#L8-L9

Oh..Ok. makes sense now Wesley. So, am I right in understanding that I will see a new screen when all changes are pushed to your beta driver?